src/__test__/e2e.test.ts

```diff
@@ -52,3 +52,62 @@ test(
   },
   { timeout: 20000 }
 );
+
+test(
+  "End to end test - separate entrypoints",
+  async () => {
+    // Initialize prisma:
+    await exec("yarn prisma init --datasource-provider mysql");
+
+    // Set up a schema
+    await fs.writeFile(
+      "./prisma/schema.prisma",
+      `datasource db {
+        provider = "mysql"
+        url      = "mysql://root:password@localhost:3306/test"
+    }
+
+    generator kysely {
+        provider  = "node ./dist/bin.js"
+        enumFileName = "enums.ts"
+    }
+
+    enum TestEnum {
+        A
+        B
+        C
+    }
+    
+    model TestUser {
+        id          String @id
+        name        String
+        age         Int
+        rating      Float
+        updatedAt   DateTime
+        abc         TestEnum
+    }`
+    );
+
+    // Run Prisma commands without fail
+    // await exec("yarn prisma db push"); -- can't push to mysql, enums not supported in sqlite
+    await exec("yarn prisma generate"); //   so just generate
+
+    const typeFile = await fs.readFile("./prisma/generated/types.ts", {
+      encoding: "utf-8",
+    });
+    expect(typeFile).not.toContain("export const");
+    expect(typeFile).toContain(`import type { TestEnum } from "./enums";`);
+
+    const enumFile = await fs.readFile("./prisma/generated/enums.ts", {
+      encoding: "utf-8",
+    });
+    expect(enumFile).toEqual(`export type TestEnum = "A" | "B" | "C";
+export const TestEnum = {
+  A: "A",
+  B: "B",
+  C: "C",
+};
```

Comment:
separate issue but this should ideally be an `as const` object

Reply:
Agreed!